### PR TITLE
doc(roles): update privileges

### PR DESCRIPTION
**Pull request by ksrinath**
## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: docs/authorization/roles.md [hunk 1]
```diff
@@ -79,69 +79,93 @@ These privileges are common to both Self-Hosted DataHub and Managed DataHub.
...
+| Edit Owners                        | :heavy_check_mark: | :x:                | :x:                | The ability to add and remove owners of an entity.                                               |
+| Edit Group Members                 | :heavy_check_mark: | :x:                | :x:                | The ability to add and remove members to a group.                                                |
+| Edit User Profile                  | :heavy_check_mark: | :x:                | :x:                | The ability to change the user's profile including display name, bio, title, profile image, etc. |
+| Edit Contact Information           | :heavy_check_mark: | :x:                | :x:                | The ability to change the contact information such as email & chat handles.                      |
+| Delete                             | :heavy_check_mark: | :x:                | :x:                | The ability to delete the delete this entity.                                                    |
```
Comment (eboneil) [inline docs/authorization/roles.md:133]: "The ability to delete the delete this entity." -> "The ability to delete this entity."

File: docs/authorization/roles.md [hunk 2]
```diff
@@ -79,69 +79,93 @@ These privileges are common to both Self-Hosted DataHub and Managed DataHub.
...
+|-----------------------------|--------------------|--------------------|--------|-----------------------------------------------------------------------------------------------------|
+| Manage Tests                | :heavy_check_mark: | :heavy_check_mark: | :x:    | Create and remove Asset Tests.                                                                      |
+| View Metadata Proposals     | :heavy_check_mark: | :heavy_check_mark: | :x:    | View the requests tab for viewing metadata proposals.                                               |
+| Create metadata constraints | :heavy_check_mark: | :heavy_check_mark: | :x:    | Create metadata constraints.                                                                        |
+| Manage Platform Settings    | :heavy_check_mark: | :x:                | :x:    | View and change platform-level settings, like integrations & notifications.                         |
+| Manage Connections          | :heavy_check_mark: | :x:                | :x:    | Manage connections to external platforms.                                                           |
```
Comment (eboneil) [inline docs/authorization/roles.md:147]: How about "Manage connections to external DataHub platforms." so it's more clear that you can't connect to like snowflake or other platforms via this connection?

**Review by eboneil — COMMENTED** [review]
Overall this looks pretty good, though I did notice that this page is out of sync with this page: https://datahubproject.io/docs/authorization/policies/
i.e., there are policies missing on one that are on the other. Can we get them into parity with this PR as well, since my customers check both places for guidance?

File: docs/authorization/roles.md (hunk 1 above)
Comment (ksrinath) [reply docs/authorization/roles.md]: Addressed.

File: docs/authorization/roles.md (hunk 2 above)
Comment (ksrinath) [reply docs/authorization/roles.md]: Addressed.

**Pull request merged**